widget/splitcontainer.go

```diff
@@ -0,0 +1,272 @@
+package widget
+
+import (
+	"image/color"
+
+	"fyne.io/fyne"
+	"fyne.io/fyne/canvas"
+	"fyne.io/fyne/driver/desktop"
+	"fyne.io/fyne/theme"
+)
+
+func getDividerThickness() int {
+	return theme.Padding() * 2
+}
+
+func getDividerLength() int {
+	return theme.Padding() * 6
+}
+
+func getHandleThickness() int {
+	return theme.Padding() / 2
+}
+
+func getHandleLength() int {
+	return theme.Padding() * 4
+}
+
+// Declare conformity with interfaces
+var _ fyne.CanvasObject = (*divider)(nil)
+var _ fyne.Draggable = (*divider)(nil)
+var _ desktop.Hoverable = (*divider)(nil)
+
+type divider struct {
+	BaseWidget
+	split   *SplitContainer
+	hovered bool
+}
+
+func (d *divider) DragEnd() {
+}
+
+func (d *divider) Dragged(event *fyne.DragEvent) {
+	offset := d.split.offset
+	if d.split.Horizontal {
+		offset += event.DraggedX
+	} else {
+		offset += event.DraggedY
+	}
+	d.split.updateOffset(offset)
+}
+
+func (d *divider) MouseIn(event *desktop.MouseEvent) {
+	d.hovered = true
+	d.split.Refresh()
+}
+
+func (d *divider) MouseOut() {
+	d.hovered = false
+	d.split.Refresh()
+}
+
+func (d *divider) MouseMoved(event *desktop.MouseEvent) {}
+
+// CreateRenderer is a private method to Fyne which links this widget to its renderer
+func (d *divider) CreateRenderer() fyne.WidgetRenderer {
+	d.ExtendBaseWidget(d)
+	r := canvas.NewRectangle(theme.IconColor())
```

Comment:
This colour will also not reflect theme changes - should update in Refresh()

Reply:
Ack